charts/vsphere-csi/v2.1.0/templates/vsphere-csi-secret.yaml

```diff
@@ -1,21 +1,9 @@
+{{- if .Values.vCenter.generateCsiVSphereConfSecret -}}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: vsphere-config-secret
+  name: {{ .Values.vCenter.csiVSphereConfSecretName }}
   namespace: {{ .Release.Namespace }}
-stringData:
-  csi-vsphere.conf: |
-    [Global]
-    cluster-id = {{ uuidv4 }}
-    user = {{ .Values.vCenter.username | quote }}
-    password = {{ .Values.vCenter.password | quote }}
-    port = {{ .Values.vCenter.port | default "443" | quote }}
-    insecure-flag = {{ .Values.vCenter.insecureflag | default "1" | quote }}
-
-    [VirtualCenter {{ .Values.vCenter.host | quote }}]
-    datacenters = {{ .Values.vCenter.datacenters | quote }}
-
-    [Workspace]
-    server = {{ .Values.vCenter.host | quote }}
-    datacenter = {{ .Values.vCenter.datacenter | quote }}
-    default-datastore = {{ .Values.vCenter.defaultDatastore | quote }}
```

Comment:
This seems to be redundant so I removed it. It doesn't even seem to be in their config type [here](https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/common/config/types.go).